org.eclipse.scout.rt.platform/src/main/java/org/eclipse/scout/rt/platform/util/XmlUtility.java

```diff
@@ -492,8 +505,24 @@ public static Document getXmlDocument(URL url) {
    *     if there is an error parsing the content of the {@link String} into a {@link Document}.
    */
   public static Document getXmlDocument(String rawXml) {
+    return getXmlDocument(rawXml, false);
+  }
+
+  /**
+   * Creates an xml {@link Document} filled with the content of the given {@link String}.
```

Comment:
"xml" should be uppercase in regular texts (multiple occurrences in this javadoc).